docs/how-shipkit-works.md

```diff
@@ -4,111 +4,22 @@ Want to find out how Shipkit works?
 - see also [getting started guide](/docs/getting-started.md)
 - or the entire [documentation index](/README.md#documentation)
 
-### Version bumps
+### High level
 
-Yesterday version is already old.
-Frequent, automated releases need some way to manage incrementing versions.
-Welcome to continuous delivery!
+TODO write here about Gradle, shipkit file and refer to the getting started guide
 
-#### Intro
+### Details
 
-Shipkit needs to be able to automatically use the right version when building and publishing.
-This way, you don't need to deal with changing/updating the version for every release.
-Every release will produce n+1 version.
+How do we:
+- [bump versions](/docs/features/version-bumps.md)
+- [generate release notes](/docs/features/automated-release-notes.md)
+- [publish binaries](/docs/gradle-plugins/publishing-binaries.md)
```

Comment:
This link seems to be broken. It should rather be `docs/features/publishing-binaries.md`, isn't it?